FlowCrypt/Controllers/Compose/ComposeViewController.swift

```diff
@@ -117,24 +121,46 @@ final class ComposeViewController: TableNodeViewController {
             appDelegateGoogleSessionContainer: UIApplication.shared.delegate as? AppDelegate,
             shouldRunWarmupQuery: true
         )
-        self.composeMessageService = composeMessageService ?? ComposeMessageService(
-            appContext: appContext,
-            keyMethods: keyMethods
-        )
+        let draftGateway = try appContext.getRequiredMailProvider().draftGateway
+
+        if let composeMessageService = composeMessageService {
+            self.composeMessageService = composeMessageService
+        } else {
+            self.composeMessageService = try await ComposeMessageService(
+                appContext: appContext,
+                keyMethods: keyMethods,
+                draftGateway: draftGateway
+            )
+        }
```

Comment:
I think you updated above code because `ComposeMessageService` requires  `messageOperationsProvider` (and it needs to be async/await)
But As I checked, `ComposeMessageService.messageOperationsProvider` is not used at all and we can remove that and revert above change to below one?

   ```swift
     self.composeMessageService = composeMessageService ?? ComposeMessageService(
            appContext: appContext,
            keyMethods: keyMethods
        )

```